### PR TITLE
feat(api-service): DDOG labels + probes improvement

### PR DESCRIPTION
**Pull request by tadam313**
* ddog labels + env variables
* probes customization
* run as non-root user

### CODE REVIEW

File: charts/api-service/values.yaml [hunk 1]
```diff
@@ -4,23 +4,42 @@
...
 
 image:
-  repository: sethvargo/envserver
+  name: "sethvargo/envserver:latest"
   pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: "latest"
+  # Berglas integration requires an image command
+
+healthCheck:
```
Comment (zsolt-marta-bitrise) [inline charts/api-service/values.yaml:29]: I think we should name this `livenessCheck` so it's clear what it refers to exactly

File: charts/api-service/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Api service chart
+
+A simple helm chart to run **stateless** API services. Features:
+
+* automatic Datadog logging and APM (also via istio) integration
+* autoscaling
```
Comment (zsolt-marta-bitrise) [inline charts/api-service/README.md:6]: This is not yet enabled: https://github.com/bitrise-io/k8s-recipes/pull/6/files#diff-895562ea42547d61698c301a970f211f8a5a27ea79ce4d6b82af95f0077763acL65

Comment (zsolt-marta-bitrise) [reply charts/api-service/README.md:6]: I'd rather put this to the future work section

Comment (tadam313) [reply charts/api-service/README.md:6]: I know but it _can_ be enabled

**Review by zsolt-marta-bitrise — APPROVED** [review]

**Pull request merged**